grafana-plugin/src/utils/index.ts

```diff
@@ -27,7 +27,7 @@ export function showApiError(error: any) {
 }
 
 export function refreshPageError(error: any) {
-  if (error.response.status == 502) {
+  if (error.response.status === 502) {
```

Comment:
@maskin25 tagging you here based on our conversation on Slack. 

this should be a safe change. `refreshPageError` is currently only used [here](https://github.com/grafana/oncall/blob/dev/grafana-plugin/src/models/alertgroup/alertgroup.ts#L260) to catch potential errors from `axios`. According to `axios`'s [types](https://github.com/axios/axios/blob/9bd53214f6339c3064d4faee91c223b35846f2dd/index.d.ts#L342), `error.response.status` will always be a `number`.

I will also go ahead and update the `any` here to the proper `AxiosResponse` type